### PR TITLE
feat: Export event as iCalendar

### LINKED ISSUE #4811: Add feature to export event as iCalendar
**Is your feature request related to a problem? Please describe.**
Add the API endpoint to export event as iCalendar.
 
**Describe the solution you'd like**
Follow the process defined in v1 - open-event-legacy for the task and the export event as a zip endpoint in v2 for the API.

**Describe alternatives you've considered**
None

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #4811 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Adds feature to export event as iCalendar

#### Changes proposed in this pull request:
- Followed the method used in version 1.

### CODE REVIEW

**Comment by mayank8318** [comment]
@iamareebjamal @niranjan94 @poush @srv-twry @bhaveshAn Please review

**Comment by Kreijstal** [comment]
Travis build is failing

**Comment by mayank8318** [comment]
@Kreijstal It is failing for `/v1/events/1/export/json` I don't know why. It is passing for the new route though

**Comment by niranjan94** [comment]
@mayank8318 doesn't matter which test is failing. If a test passes on `development` but fails on your branch, there is a good chance it was as a result of changes done here. So, please investigate and fix.

**Comment by mayank8318** [comment]
@Kreijstal @niranjan94 Please review now :)

File: app/api/helpers/ICalExporter.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                                                   identifier=event.identifier, _external=True))
+
+                for speaker in session.speakers:
+                    # Ref: http://icalendar.readthedocs.io/en/latest/usage.html#file-structure
+                    # can use speaker.email below but privacy reasons
+                    attendee = vCalAddress('MAILTO:' + event.email if event.email else 'undefined@email.com')
+                    attendee.params['cn'] = vText(speaker.name)
+                    event_component.add('attendee', attendee)
+
+                cal.add_component(event_component)
```
Comment (iamareebjamal) [inline app/api/helpers/ICalExporter.py]: There is too much nesting, use early return instead

Comment (mayank8318) [reply app/api/helpers/ICalExporter.py]: @iamareebjamal Where do you recommend I use early return cause I need to loop through all the items right?

Comment (iamareebjamal) [reply app/api/helpers/ICalExporter.py]: Then use continue

Comment (mayank8318) [reply app/api/helpers/ICalExporter.py]: But it is a direct for-loop with a just a primary if condition?

Comment (iamareebjamal) [reply app/api/helpers/ICalExporter.py]:
```
for :
  if not x:
    continue
  do stuff
```

**Pull request merged**